backtester/eventhandlers/portfolio/risk/risk.go

```diff
@@ -0,0 +1,68 @@
+package risk
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/thrasher-corp/gocryptotrader/backtester/eventhandlers/portfolio/compliance"
+	"github.com/thrasher-corp/gocryptotrader/backtester/eventhandlers/portfolio/holdings"
+	"github.com/thrasher-corp/gocryptotrader/backtester/eventtypes/order"
+	"github.com/thrasher-corp/gocryptotrader/currency"
+)
+
+// EvaluateOrder goes through a standard list of evaluations to make to ensure that
+// we are in a position to follow through with an order
+func (r *Risk) EvaluateOrder(o order.Event, latestHoldings []holdings.Holding, s compliance.Snapshot) (*order.Order, error) {
+	if o == nil || latestHoldings == nil {
+		return nil, errors.New("received nil argument(s)")
+	}
+	retOrder := o.(*order.Order)
+	if o.IsLeveraged() {
+		if !r.CanUseLeverage {
+			return nil, errors.New("order says to use leverage, but it is not allowed")
+		}
+		ratio := existingLeverageRatio(s)
+		lookupRatio := r.CurrencySettings[o.GetExchange()][o.GetAssetType()][o.Pair()].MaxLeverageRatio
```

Comment:
We can move this call out of the if statement and use it once:
lookup, ok := r.CurrencySettings[o.GetExchange()][o.GetAssetType()][o.Pair()]